selfdrive/ui/qt/onroad.cc

```diff
@@ -162,6 +172,108 @@ void OnroadAlerts::paintEvent(QPaintEvent *event) {
   }
 }
 
+// OnroadHud
+OnroadHud::OnroadHud(QWidget *parent) : QWidget(parent) {
+  engage_img = QPixmap("../assets/img_chffr_wheel.png").scaled(img_size, img_size, Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

Comment:
let's make a helper for this in qt utils. we do this in quite a few places

Reply:
think it's best if the helper and using it is its own PR

Reply:
i was thinking about this :D